### PR TITLE
post: gradle kotlin DSL contribution

### PR DESCRIPTION
**Pull request by jnizet**
Scheduled for Tuesday

### CODE REVIEW

File: _posts/2018-09-18-gradle-kotlin-dsl-documentation.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+So you have no excuse anymore. 
+Try the Kotlin DSL. 
+It works, it is quite close to the Groovy DSL, but with less black magic involved, and it does allow auto-completion and navigation to the sources in IntelliJ.
+
+Translating the samples has been a great experience. 
+Contributing to an open-source project you like and respect is always gratifying. 
+You get the feeling that what you're doing matters. 
+Gradle folks have been nothing but kind, understanding, helpful, grateful... and demanding. 
+And it helped finding and fixing a few issues, too.
```
Comment (clacote) [inline _posts/2018-09-18-gradle-kotlin-dsl-documentation.md]: Not sure what “it” is? The Gradle folks (but that should be “they”)? Your contribution to Gradle?

Comment (jnizet) [reply _posts/2018-09-18-gradle-kotlin-dsl-documentation.md]: Yes, it's the translation to Kotlin. I'll rephrase and reorder.

File: _posts/2018-09-18-gradle-kotlin-dsl-documentation.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+You get the feeling that what you're doing matters. 
+Gradle folks have been nothing but kind, understanding, helpful, grateful... and demanding. 
+
+I didn't just decide to contribute though. 
+That's always intimidating: where to start? 
+How to get help? 
+Will I help, or will I be a burden for the maintainers?
+
+I contributed because the Gradle team asked me to. 
+The first time was after I wrote my migration guide, and then by opening [this epic issue](https://github.com/gradle/gradle/issues/6442), asking for help from contributors, and providing detailed instructions and examples on how to accomplish the task.
```
Comment (cexbrayat) [inline _posts/2018-09-18-gradle-kotlin-dsl-documentation.md]: I understood the sentence as if you were the author of the epic issue. Maybe ".. migration guide, and when they opened this epci issue"?

File: _posts/2018-09-18-gradle-kotlin-dsl-documentation.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+I wish more big open-source projects do that. 
+Tagging issues with "ideal-for-contribution" is also nice.
+What might seem like grunt work for project maintainers or experienced contributors is an interesting challenge and learning experience for casual, less-experienced developers who are willing to help.
+
+So, if you're an open-source project maintainer and you read this, please make it easy to start contributing on your project. 
+Ask for help. 
+And communicate on public channels (blogs, tweets, etc.) about it.
```
Comment (cexbrayat) [inline _posts/2018-09-18-gradle-kotlin-dsl-documentation.md]: Maybe worth adding a few links on the popular iniatives out there like https://www.firsttimersonly.com/ https://yourfirstpr.github.io/ https://up-for-grabs.net/

Comment (cexbrayat) [reply _posts/2018-09-18-gradle-kotlin-dsl-documentation.md]: Unrelated but we should also add a CONTRIBUTING.md file on our open source projects...

**Pull request merged**